### DISCUSSION TITLE
Is there a way to ignore / turn off guidance boundary in Quest 2?

### DISCUSSION THREAD

**Discussion by leeenglestone** [StereoKit/StereoKit#397] [Q&A] [answered]
Apologies if this is easy, but if I want to create a VR experience that runs on the Quest 2  that requires me to move around a lot is there an easy wat to turn off warning boundaries/pass through that appears when you approach the boundary?

**Comment by maluoi** [comment, marked as answer]
Yeah! This is an Oculus OS safety feature, so I don't believe there's a programmatic way of doing it. Would be kinda interested to know if there was! You can turn it off in the settings menu at `Settings->System->Developer->Guardian`

**Reply by leeenglestone** [reply]
Yep found it thanks
